### ISSUE TITLE
Client watch mode enabled but not working

### ISSUE THREAD

**Issue by tebeco** [fuse-box/fuse-box-examples#1]
# TL;DR
### /!\ Change of server bundle (like `index.html`) works on simple F5

# Repro steps
```
> git clone https://github.com/fuse-box/preview-1.4.1
> cd .\preview-1.4.1\examples\server-client
> node fuse.js
```

content of client part of fuse.js :
```
fuse.bundle("client/app")
    .watch("client/**") // watch only client related code
    .hmr()
    .instructions(" > client/index.ts");
```

```
└── default (1 files,  553 Bytes)
      server/index.js

    Bundle
    Size: 553 Bytes
    Time: 29ms
  → Typescript config file was not found. Improvising
└── default (1 files,  340 Bytes)
      client/index.js
└── fusebox-hot-reload@0.0.0 (0 files) 1.8 kB
└── fusebox-websocket@0.0.0 (0 files) 2.9 kB
└── events@0.0.0 (0 files) 9.8 kB
└── path@0.0.0 (0 files) 6 kB
└── process@0.0.0 (0 files) 3 kB
└── object-assign-polyfill@0.0.0 (0 files) 1 kB

    Bundle
    Size: 24.8 kB
    Time: 327ms
14:54:32 : Launching socket server on 4445
Powered by FuseBox
Example app listening on port 3000!
14:54:33 : HMR is enabled
```

* Open a browser on `http://localhost:3000/` second line is `!Some stuff here!! a/b/c`
* Go to `\preview-1.4.1\examples\server-client\client\index.ts` and change :
`h1.innerHTML = "!Some stuff here!!" + path.join("a", "b", "c")`
to 
`h1.innerHTML = "!Some stuff here!!" + path.join("dfsjkghdghkgh", "b", "c")`

* Save => Nothing change
* F5 => Nothing change
* Ctrl + F5 => Nothing change
* Kill Node & restart => Finally working

as far as i understand fuse.js :

```
fuse.bundle("client/app")
    .watch("client/**") // watch only client related code
    .hmr()
    .instructions(" > client/index.ts");
```

it should watch that `client\index.ts`, something seems broken right ?

/!\ ## Change of index.html works on simple F5

**Comment by devmondo** [comment]
@tebeco  `1.4.1` is in preview now, there is a bug on the watch, please use watch without passing any params `watch()` it should work for now.
